packages/core/src/components/ButtonGroup/__stories__/buttonGroup.stories.tsx

```diff
@@ -252,3 +252,34 @@ export const ButtonGroupAsToggle: Story = {
   ),
   name: "Button group as toggle"
 };
+
+export const StretchedButtonGroup: Story = {
```

Comment:
Design want to maybe throw it in a smaller container, like a DialogContentContainer or something

Reply:
Dialog Content Container - doesn't have a fixed width, its width is according to its content/parent - in addition it has   box shadow.

I will give it the following -  `width: clamp(200px, 40%, 400px);`

Reply:
<img width="1099" alt="image" src="https://github.com/mondaycom/vibe/assets/60314759/feb25f7a-7e96-4657-b1f0-d330b7213f70">

changed it to `width: clamp250px, 100%, 750px);`